velox/connectors/hive/tests/HivePartitionFunctionTest.cpp

```diff
@@ -261,11 +261,12 @@ TEST_F(HivePartitionFunctionTest, timestamp) {
 }
 
 TEST_F(HivePartitionFunctionTest, date) {
-  auto values = makeNullableFlatVector<Date>(
+  auto values = makeNullableFlatVector<int32_t>(
```

Comment:
This test can be removed I think since partitioning by the Date struct is not needed anymore.